cmake/compiler/gcc/compiler_flags.cmake

```diff
@@ -111,6 +111,12 @@ if (NOT CONFIG_NEWLIB_LIBC AND
   set_compiler_property(APPEND PROPERTY nostdinc_include ${NOSTDINC})
 endif()
 
+if (CONFIG_PICOLIBC AND
+    NOT CONFIG_PICOLIBC_IO_FLOAT)
+  set_compiler_property(PROPERTY no_printf_return_value -fno-printf-return-value)
```

Comment:
there needs to be an entry for `no_printf_return_value` in `cmake/compiler/compiler_flags_template.cmake`.

Reply:
thanks, added.